src/pages/ErrataFeed.vue

```diff
@@ -65,6 +65,51 @@
         binary-state-sort
         wrap-cells
       >
+        <template v-slot:top-right>
+          <q-btn
+            size="80%"
+            no-caps
+            icon="restart_alt"
+            color="grey-8"
+            @click="showDialogAdvisories = true"
+            :loading="loadingReset"
+          >
+            Reset
+            <q-tooltip
+              >Reset matched packages of erratas issued after a specified date
+            </q-tooltip>
+          </q-btn>
+          <q-dialog v-model="showDialogAdvisories">
+            <q-card>
+              <q-card-section>
+                <div class="text-h6">Reset Packages</div>
+                <br />
+                <div>
+                  Reset matched packages of Advisories after a specified date:
+                </div>
+                <div
+                  class="q-pb-md group row justify-center"
+                  style="gap: 10px; margin: 10px"
+                >
+                  <input
+                    class="date"
+                    type="datetime-local"
+                    style="
```

Comment:
Use the style section below to define a new class and set it to the class section for the element